styles/globals.css

```diff
@@ -1,35 +1,11 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 html,
 body {
   padding: 0;
```

Comment:
Can't we also do this with tailwind classes, for both padding 0 and margin 0?

Reply:
Yes we can but can find the HTML and body elements in nextjs